src/components/containers/victory-brush-container.js

```diff
@@ -116,8 +115,7 @@ export default class VictoryBrushContainer extends VictoryContainer {
   // Overrides method in VictoryContainer
   getChildren(props) {
     const children = React.Children.toArray(props.children);
-    const components = [...children, this.getRect(props)];
-    return components.map((component, i) => {
+    return map([...children, this.getRect(props)], (component, i) => {
```

Comment:
While we're on the subject, lodash `concat` is faster. `_.concat(children, this.getRect(props)`